serverless/tensorflow/matterport/mask_rcnn/nuclio/model_loader.py

```diff
@@ -63,7 +63,7 @@ def infer(self, image, threshold):
                 contour = np.flip(contour, axis=1)
                 # Approximate the contour and reduce the number of points
                 contour = approximate_polygon(contour, tolerance=2.5)
-                if len(contour) < 3:
+                if len(contour) < 5:
```

Comment:
It might be better to use 6 here, though.

Reply:
Agreed. Updated.